src/main/resources/vm/script/datamanagement/refresh.vm

```diff
@@ -0,0 +1,97 @@
+## refresh.vm
+
+
+shanoir_token_location="${PWD}/cache/SHANOIR_TOKEN.txt"
+shanoir_refresh_token_location="${PWD}/cache/SHANOIR_REFRESH_TOKEN.txt"
+refreshingJobStarted=false
+
+REFRESH_PID=""
+
+#
+# this is a background process to refresh shanoir token
```

Comment:
Maybe add here for documentation an exemple of the expected URI with the mandatory arguments